NewCode/view.py

```diff
@@ -0,0 +1,9 @@
+def view_data(*data):
+    id, name, lastname, phone = data
+    result = ''
+    print('id name_user lastname_user phone_number')
+    for i in id:
+        result = str(id[i])+' '+str(name[i])+' '+str(lastname[i])+\
+            ' '+str(phone[i])
```

Comment:
```suggestion
                result = f'{id[i]} {name[i]} {lastname[i]} {phone[i]}'
```
Мне кажется, так намного проще.